tools/README.md

```diff
@@ -84,7 +84,12 @@ python3 /graphstorm/tools/partition_graph.py --dataset ogbn-arxiv \
                                              --is-homo     
 ```
 
-## Convert features from distDGL format to WholeGraph format
+## Use WholeGraph to accelerate training and inferencing
+Graphstorm leverages NVIDIA’s [Wholegraph](https://github.com/rapidsai/wholegraph) framework to efficiently transfer node and edge features between machines. This capability can substantially enhance the speed of both training and inferencing pipelines. To take advantage of this feature, users are required to have EFA network support on their cluster. For a step-by-step setup guide, please refer to the [tutorial](https://graphstorm.readthedocs.io/en/latest/advanced/advanced-wholegraph.html). Converting node and edge features to the WholeGraph format is the only manual step; the rest of the process is seamless. 
+
+Please not, we do not support conversion of `train_mask`, `test_mask`, `val_mask` or `labels` to WholeGraph format. 
```

Comment:
```suggestion
Please note, we do not support conversion of `train_mask`, `test_mask`, `val_mask` or `labels` to WholeGraph format. 
```

Reply:
We also need to mention that other than masks and labels, all other node and edge features must be converted into WholeGraph format using `convert_feat_to_wholegraph.py` toolkit.